modules/secured-cluster-configuration-options-operator.adoc

```diff
@@ -167,9 +167,11 @@ These components are Collector and Compliance.
 | Parameter | Description
 
 | `perNode.collector.collection`
-| The method for system-level data collection. The default value is `EBPF`. Red Hat recommend using `EBPF` as the value for this parameter.
-If you select `NoCollection`, you will not be able to see any information about network activity and process executions.
-Options include `NoCollection`, `EBPF`, and `KernelModule`.
+| The method for system-level data collection.
+The default value is `EBPF`.
+Red Hat recommends using `EBPF` for data collection.
+If you select `NoCollection`, you will not see any information about the network activity and the process executions.
```

Comment:
```suggestion
If you select `NoCollection`, then any information about the network activity and the process executions is not available.
```
Just a suggestion. wdyt ?

Reply:
Thank you @darshan-nagaraj 

🤔 Yeah maybe I could rewrite it as 
```
If you select NoCollection, Collector does not report any information about the 
network activity and the process executions.
```